lib/wab/client.rb

```diff
@@ -43,6 +43,7 @@ def initialize(addr, port, options={})
     # query:: query parameters to match against existing instances. A match fails the insert.
     def create(data, kind=nil, query=nil)
       kind = data[@type_key] if kind.nil?
+      kind = data[@type_key.to_s] if kind.nil?
```

Comment:
Would the following work instead of the two lines with nil-check?
```ruby
kind ||= (data[@type_key] || data[@type_key.to_s])
```

Reply:
It would.